Applications of Spark.md

```diff
@@ -0,0 +1,2 @@
+# Applications of Spark
```

Comment:
I would like to suggest find some examples of applications from Russian companies and proejcts.
What do you think?

Reply:
Issues added, branchnames changed, info about Spark in Russia added. Check new pull requests, please